src/button-layout.tsx

```diff
@@ -139,6 +139,7 @@ const ButtonLayout: React.FC<ButtonLayoutProps> = ({
         <div
             ref={wrapperElRef}
             className={classnames(
+                styles.baseContainer,
```

Comment:
while the layout is in the "measuring state", we also need the `display: flex`, `margin`, `flexWrap` styles